### PR TITLE
DOC: ran blacken docs tool and checked output to improve formatting #36777

### PR DESCRIPTION
**Pull request by maria-ilie**
For task #36777
Ran blacken-tools on user_guide/10min.rst, user_guide/advanced.rst, basics.rst

### CODE REVIEW

**Comment by MarcoGorelli** [comment]
Thanks @maria-ilie ! This looks correct, it's just that E203 needs to be added to the ignore list for [flake8-rst]

Could you make the following change to `setup.cfg` as well? I think it's fine if it's part of this same pull request

```diff
diff --git a/setup.cfg b/setup.cfg
index 73986f692..eb2efb937 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -40,7 +40,9 @@ bootstrap =
     np  # avoiding error when importing again numpy or pandas
     pd  # (in some cases we want to do it to show users)
 ignore = E402,  # module level import not at top of file
+         E203,  # space before : (needed for how black formats slicing)
          W503,  # line break before binary operator
+         W504,  # line break after binary operator
          # Classes/functions in different blocks can generate those errors
          E302,  # expected 2 blank lines, found 0
          E305,  # expected 2 blank lines after class or function definition, found 0
```

**Comment by erfannariman** [comment]
Just to be sure we don't get any conflicts, I edited `10min.rst` on #36780

**Comment by dsaxton** [comment]
Thanks @maria-ilie for the PR, can you merge master to fix the conflict?

File: doc/source/user_guide/10min.rst [hunk 1]
```diff
@@ -667,9 +667,10 @@ pandas can include categorical data in a :class:`DataFrame`. For full docs, see
 
 .. ipython:: python
 
-   df = pd.DataFrame(
-       {"id": [1, 2, 3, 4, 5, 6], "raw_grade": ["a", "b", "b", "a", "a", "e"]}
-   )
+    df = pd.DataFrame(
+        {"id": [1, 2, 3, 4, 5, 6], "raw_grade": ["a", "b", "b", "a", "a", "e"]}
+    )
```
Comment (MarcoGorelli) [inline doc/source/user_guide/10min.rst:672]: Looks like this block got extra-indented by one space, this is causing the CI error

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks for updating, there's just one small thing which is causing the automated checks to fail

File: doc/source/user_guide/basics.rst [hunk 2]
```diff
@@ -795,14 +803,13 @@ Now compare the following:
...
-    add_country_name(extract_city_name(df_p), country_name='US')
+    add_country_name(extract_city_name(df_p), country_name="US")
 
 Is equivalent to:
 
 .. ipython:: python
 
-    (df_p.pipe(extract_city_name)
-         .pipe(add_country_name, country_name="US"))
+    (df_p.pipe(extract_city_name).pipe(add_country_name, country_name="US"))
```
Comment (dsaxton) [inline doc/source/user_guide/basics.rst:812]:
```suggestion
    df_p.pipe(extract_city_name).pipe(add_country_name, country_name="US")
```

**Review by dsaxton — APPROVED** [review]
Looks good, just one tiny nitpick. Also merging master should fix the CI.

**Comment by dsaxton** [comment]
Thanks @maria-ilie

**Comment by maria-ilie** [comment]
Thanks so much for reviewing!

**Comment by MarcoGorelli** [comment]
Happy to help, let us know if you'd like any help/mentoring with other issues

**Pull request merged**